### DISCUSSION TITLE
I need a composer!

### DISCUSSION THREAD

**Discussion by DabbinDude316B** [FunkinCrew/Funkin#1977] [Q&A] [answered]
(Use soft girl voice for Opponent)

Song 1: Slow, Calm and nothing really happens
Song 2: Super slow, but gets suddenly fast near the end.
Song 3: A mix of Slow, Fast, And super fast
Song 4: Fast and Super fast song

**Comment by itsDARK768** [comment, marked as answer]
This is the wrong place to ask that, please go ask someone on discord that is a composer.

**Reply by Beepb0bb00p** [reply]
true

**Comment by DabbinDude316B** [comment]
Oh ok
